src/MappingOperation/Implementations/MapToMultiple.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace AutoMapperPlus\MappingOperation\Implementations;
+
+use AutoMapperPlus\Exception\UnregisteredMappingException;
+use AutoMapperPlus\MappingOperation\ContextAwareOperation;
+use AutoMapperPlus\MappingOperation\ContextAwareTrait;
+use AutoMapperPlus\MappingOperation\DefaultMappingOperation;
+use AutoMapperPlus\MappingOperation\MapperAwareOperation;
+use AutoMapperPlus\MappingOperation\MapperAwareTrait;
+use Traversable;
+use function is_array;
+
+/**
+ * Class MapTo.
+ * Allows a property to be mapped itself.
+ * @package AutoMapperPlus\MappingOperation\Implementations
+ */
+class MapToMultiple extends DefaultMappingOperation implements
```

Comment:
It seems you didn't include a `mapToMulitiple` method in the `Operation` class; feel free to do so.

Reply:
added